tests/test-proxy.js

```diff
@@ -138,13 +138,13 @@ function addTests() {
       url    : 'https://google.com',
       tunnel : false,
       pool   : false
-    }, true)
+    }, false)
```

Comment:
why are we no longer using a proxy for these requests? Shouldn't this change require no change to testing?